address_map.txt

```diff
@@ -118,11 +124,54 @@ declare     tpage                   u8
 declare     clut                    u8
 declare     nFloor                  u8
 declare     parts_num               u8
-declare     pKan_t_ptr              s32
+declare     pKan_t_ptr              void*
 declare     water                   s16
 declare     type                    u8
 declare     var_10F                 u8
 
+struct      EnemyEntity
```

Comment:
I think we know lot's of these variables. So we should be able to rename them.

Reply:
I am wondering whether PlayerEntity and EnemyEntity need to inherit a new shared base class.

Reply:
I think so, before introducing interop/address_map it was like that, right?